.github/workflows/release.yml

```diff
@@ -127,9 +127,15 @@ jobs:
 
           $(go env GOPATH)/bin/gox -ldflags "$($(go env GOPATH)/bin/ziti-ci -q -t go-build-flags)" -cgo -os=linux -arch=amd64 -output=$GOX_OUTPUT ./ziti/
 
+          GOEXPERIMENT=boringcrypto \
+          $(go env GOPATH)/bin/gox -ldflags "$($(go env GOPATH)/bin/ziti-ci -q go-build-flags -n)" -cgo -os=linux -arch=amd64 -output="${GOX_OUTPUT}-boring" ./ziti/
+
           CC=arm-linux-gnueabihf-gcc \
           $(go env GOPATH)/bin/gox -ldflags "$($(go env GOPATH)/bin/ziti-ci -q -t go-build-flags)" -cgo -os=linux -arch=arm -output=$GOX_OUTPUT ./ziti/
 
+          GOEXPERIMENT=boringcrypto \
```

Comment:
The arm64 build is sourced for arm64 container images. This enables the boring container images to remain deployable to the same target platform architectures as the non-boring images for Docker and Kubernetes, e.g., K3S on a RasPi would work!